rubicon-htb.js

```diff
@@ -93,6 +93,8 @@ function RubiconModule(configs) {
 
     var __pageFirstPartyData;
 
+    var _digiTrustId;
```

Comment:
is this needed as a global?  lit's only used in the `_getDigiTrustQueryParams()` function, and not sure it's needed as a global to save state between function calls (but I'm not sure).  see later comments.

Reply:
I was creating it as global in case another function or publisher want to use it on the page but I could remove it and keep it in the function scope

Reply:
OK, that makes sense as long as other pubs know of its existence.